librz/arch/isa/mips/il/fp.c

```diff
@@ -0,0 +1,314 @@
+	// Floating Point Floor Convert to Long Fixed Point
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_floor_w_fmt(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Floating Point Floor Convert to Word Fixed Point
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_ldc1(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Load Doubleword to Floating Point
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_ldxc1(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Load Doubleword Indexed to Floating Point
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_luxc1(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Load Doubleword Indexed Unaligned to Floating Point
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_lwc1(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Load Word to Floating Point
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_lwxc1(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Load Word Indexed to Floating Point
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_madd_fmt(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Floating Point Multiply Add
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_mfc1(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Move Word From Floating Point
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_mfhc1(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Move Word From High Half of Floating Point Register
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_mov_fmt(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Floating Point Move
+	// TODO: handle double
+	return SETG(REG(0), MIPS_REG(1));
+}
+
+static RzILOpEffect *mips_il_movf(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Move Conditional on Floating Point False
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_movf_fmt(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Floating Point Move Conditional on Floating Point False
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_movn_fmt(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Floating Point Move Conditional on Not Zero
+	// TODO: handle double
+	const char *rd = REG(0);
+	RzILOpPure *rs = MIPS_REG(1);
+	RzILOpPure *rt = MIPS_REG(2);
+	return BRANCH(IS_ZERO(rt), NOP(), SETG(rd, rs));
+}
+
+static RzILOpEffect *mips_il_movt(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Move Conditional on Floating Point True
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_movt_fmt(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Floating Point Move Conditional on Floating Point True
+	NOT_IMPLEMENTED;
+}
+
+static RzILOpEffect *mips_il_movz_fmt(const csh *handle, const cs_insn *insn, const ut32 gprlen) {
+	// Floating Point Move Conditional on Zero
+	// TODO: handle double
```

Comment:
Also this.

Reply:
No need. FPU stuff is not "supported" and will be properly implemented later.